build_prod.py

```diff
@@ -220,6 +220,31 @@ def build_populate_artifacts(cfg):
 
 def generate_local_conf(cfg, reconstruct_dir):
         print('Generating local.conf')
+        cfg_local_conf = cfg.get_opt_local_conf()
+        if not cfg_local_conf:
+            # if no local.conf data - make it empty
+            # the list may be filled from prod local.config
+            cfg_local_conf = []
+        # find the prod local conf
+        local_conf = cfg.get_dir_build() + '//meta-xt-{}//doc//local.conf.{}'.format(cfg.get_opt_product_type(), cfg.get_opt_product_type())
+        prod_local_conf = []
+
+        if os.path.isfile(local_conf):
+            # prod local.conf exists - filter the values,
+            # the prior is for the values from file ".cfg"
+            with open(local_conf) as conf_file:
+                for line in conf_file:
+                    delimiters = ".=","=.","+=","=+","??=","?=",":=","=" 
```

Comment:
How to handle VAR_append, VAR_remove, VAR_prepend and so on...

I agree with @andr2000, just append variables as-is without modifications. Otherwise, you could introduce potential problems which will be hard to identify.